colossalai/tensor/chunk.py

```diff
@@ -340,3 +332,17 @@ def copy_chunk_group(self, dest_group_name: str, src_group_name: str):
         for dest_chunk, src_chunk in zip(self.chunk_groups[dest_group_name], self.chunk_groups[src_group_name]):
             if not dest_chunk.is_free:
                 dest_chunk.copy_(src_chunk)
+
+    def get_chunks(self, tensors: Iterable[torch.Tensor]) -> FrozenSet[Chunk]:
+        return frozenset([self.get_chunk(tensor) for tensor in tensors])
+
+    def add_extern_static_tensor(self, tensor: torch.Tensor) -> None:
```

Comment:
I think you need to make sure the static tensor is not registered as a chunk managed tensor later.

Reply:
Yes. Done.